test/unit/dom/dom.js

```diff
@@ -200,9 +588,33 @@ describe('skatejs-named-slots dom', function () {
 
         host.innerHTML = processingInstruction;
         expect([processingInstruction, processingInstructionsAfterInnerHtml].indexOf(host.innerHTML)).to.be.above(-1);
+        expect(host.childNodes.length).to.equal(1);
+        expect(slot.getAssignedNodes().length).to.equal(1);
```

Comment:
Doesn't that mean that the test should be like this in this case: `expect(slot.getAssignedNodes().length).to.equal(0);`
and it would break?